torch/_inductor/fx_passes/post_grad.py

```diff
@@ -64,6 +63,10 @@ def post_grad_passes(gm: torch.fx.GraphModule, is_inference: bool):
         reorder_for_locality(gm.graph)
 
     if config.pattern_matcher:
+        if torch.backends.mkldnn.is_available() and config.cpp.onednn_graph:
+            from .onednn_graph_fusion import onednn_graph_fuse_fx
+
+            onednn_graph_fuse_fx(gm, is_inference)
```

Comment:
why not put this into `lazy_init`?

Reply:
I tried to put it to `lazy_init`. It requires changing its API by passing `gm` and `is_inference`. Also caused some strange errors in the unittest.  I'm checking.